src/main.rs

```diff
@@ -52,11 +78,17 @@ fn main() {
         .values_of("extra")
         .map_or_else(Vec::new, |e| e.collect());
 
-    let result = cargo_clone::clone(method, name, version, &extra);
+    let cloner = cargo_clone::Cloner::default().unwrap();
+    let result = cloner.clone(
+        cargo_clone::CloneMethodKind::from(method).unwrap(), 
```

Comment:
Can this include a comment why the `unwrap` is ok?

Reply:
Added a comment about line 83